content/tutorial-build-run-rippled-ubuntu.md

```diff
@@ -0,0 +1,173 @@
+
+For information about building `rippled` for other platforms, see [Builds](https://github.com/ripple/rippled/tree/develop/Builds) in the `rippled` GitHub repository.
+
+
+## System Requirements
+
+Ripple recommends meeting the following **minimum** system requirements for building and running `rippled`.
+
+**_To build rippled:_**
+
+* RAM: 8GB
+
+**_To run rippled:_**
+
+* RAM: 4GB
+* CPU: 64-bit x86_64, 2+ cores
+* Disk: 50GB SSD with 500 IOPS for the database partition
+* Network: A fast network connection is preferable.
+
+If you are running `rippled` in an Amazon EC2 instance, Ripple recommends selecting a **Type** of **_m3.large_**. If you choose to run a validating `rippled` server, you may need more resources.
+
+
+## 1. Build `rippled`
+
+These instructions use Ubuntu's APT (Advanced Packaging Tool) to install software prerequisites that you'll need to build `rippled`.
+
+1. Update the list of packages that can be installed or upgraded.
+```
+sudo apt-get update
+```		
+2. Retrieve and upgrade currently installed packages.
+```
+sudo apt-get -y upgrade
+```
+3. Install git.
+```
+sudo apt-get -y install git
+```
+4. Install SCons.
+```
+sudo apt-get -y install scons
+```
+5. Install Ctags.
+```
+sudo apt-get -y install ctags
+```
+6. Install `pkg-config`.
+```
+sudo apt-get -y install pkg-config
+```
+7. Install Protocol Buffers.
+```
+sudo apt-get -y install protobuf-compiler
+sudo apt-get -y install libprotobuf-dev
+```
+8. Install SSL.
+```
+sudo apt-get -y install libssl-dev
+```
+9. Install `python-software-properties`.
+```
+sudo apt-get install -y python-software-properties
+```
+10. Install Boost.
+```
+sudo add-apt-repository -y ppa:boost-latest/ppa
+sudo apt-get -y update
+sudo apt-get install -y libboost-all-dev
+```
+11. Get `rippled` source code.
+```
+git clone https://github.com/ripple/rippled.git
+cd rippled
+git checkout master
+```
+12. Build `rippled` binary executable from source code. This may take about 30 minutes.
+
+    SCons saves the built executable in `rippled/build`.
+
+```
+scons
+```
+
+
+## 2. Configure `rippled`
+
+1. Configure `rippled.cfg`.
+
+    `rippled.cfg` is the configuration file for `rippled`.
+
+    You can find an example configuration file (`rippled-example.cfg`) in `rippled/doc`. See the file for a description of all configuration options. You should be able to connect to the XRP Ledger using the default configuration defined in the example file.
+
+    Put `rippled.cfg` in `etc/opt/ripple/`.
+
+2. Configure `validators.txt`.
+
+    When you start `rippled` for the first time, it uses an initial list of validators defined in `validators.txt` to retrieve validation information and form its UNL (unique node list). `validators.txt` also enables `rippled` clients to indirectly locate IPs they can use to contact the Ripple network.
+
```

Comment:
Not sure about the reference to "Ripple network" here. What is the difference between the XRP Ledger and the Ripple network? Is the Ripple network different from RippleNet?